libs/atqr/domain/src/lib/EmailService/templates/includes/base-style.scss

```diff
@@ -0,0 +1,71 @@
+  $color_1: #272727;
```

Comment:
As a rule of thumb, in typescript is very unusual to use snake case.
In Tokilabs and in this project the use of snake case and kebab case is discouraged.
To enforce consistency in our codebase rename this, and all subsequent, variables in camelCase